01-module-system/main.js

```diff
@@ -0,0 +1,65 @@
+const help = require('./helper')
```

Comment:
please use object [destruction] (https://codeburst.io/using-destructing-for-objects-and-arrays-and-its-practical-examples-e3ebb13170e5) see first and second example